### PR TITLE
[COST-3554] Use `calculated_amortized_cost`

### PR DESCRIPTION
**Pull request by samdoran**
## Jira Ticket

[COST-3554](https://issues.redhat.com/browse/COST-3554)

## Description

Use `calculated_amortized_cost` in the settings and serializer.

Depends on #4378.

## To Do

- [x] Write data migration to change existing settings value in `user_settings` table.

### CODE REVIEW

**Comment by samdoran** [comment]
IQE test failures are related to these changes. Fixing those tests.

File: koku/api/report/constants.py [hunk 1]
```diff
@@ -10,3 +10,8 @@
 AND_AWS_CATEGORY_PREFIX = "and:aws_category:"
 OR_AWS_CATEGORY_PREFIX = "or:aws_category:"
 URL_ENCODED_SAFE = "[]:"
+AWS_MARKUP_COST = {
```
Comment (esebesto) [inline koku/api/report/constants.py:13]: Is it intentional, that we don't have "blended_cost" here?

Comment (samdoran) [reply koku/api/report/constants.py:13]: This is incorrect. Thank for catching this!

**Comment by esebesto** [comment]
I run full smokes locally with IQE branch and got a clean pass

**Review by lcouzens — APPROVED** [review]

File: koku/api/report/aws/openshift/query_handler.py [hunk 2]
```diff
@@ -122,11 +123,15 @@ def __init__(self, parameters):
...
-        )
+        kwargs = {
+            "provider": self.provider,
+            "report_type": parameters.report_type,
+            "cost_type": parameters.cost_type,
+        }
+        if markup_cost := AWS_MARKUP_COST.get(parameters.cost_type):
+            kwargs["markup_cost"] = markup_cost
+
+        self._mapper = OCPAWSProviderMap(**kwargs)
```
Comment (maskarb) [inline koku/api/report/aws/openshift/query_handler.py:134]: These changes are totally fine with me, but we need to ensure we update the other `_mapper` instantiations too for the sake of consistency. That definitely doesn't need to happen in this PR.

File: koku/api/user_settings/settings.py [hunk 3]
```diff
@@ -26,11 +25,12 @@
 ]
 
 """Default users settings"""
-USER_SETTINGS = {"settings":
-    {
-        'currency': KOKU_DEFAULT_CURRENCY,
-        'cost_type': KOKU_DEFAULT_COST_TYPE
-    }
+# fmt: off
```
[the remark this answers is not in the document]
Comment (samdoran) [reply koku/api/user_settings/settings.py:28]: It was turned off for the whole thing. I narrowed it down to just the part that black was mangling.

**Pull request merged**